src/Spryker/Client/RabbitMq/RabbitMqConfig.php

```diff
@@ -276,4 +276,12 @@ public function isDynamicStoreEnabled(): bool
     {
         return (bool)getenv('SPRYKER_DYNAMIC_STORE_MODE');
     }
+
+    /**
+     * @return int
```

Comment:
I think that `api` and comment is missing

Reply:
added